docs/build/manifest.md

```diff
@@ -5,39 +5,86 @@ The Manifest `project.yaml` file can be seen as an entry point of your project a
+  query:
+    name: '@subql/query'
+    version: 0.24.0
+description: ''
+repository: https://github.com/subquery/subql-starter
+schema:
+  file: ./schema.graphql
+network:
+  chainId: '0x91b171bb158e2d3848fa23a9f1c25182fb8e20313b2c1eb49219da7a70ce90c3'
+  endpoint: wss://polkadot.api.onfinality.io/public-ws
+  dictionary: https://api.subquery.network/sq/subquery/dictionary-polkadot
+dataSources:
+  - kind: substrate/Runtime
+    startBlock: 1
+    mapping:
+      file: "./dist/index.js"
+      handlers:
+        - handler: handleBlock
+          kind: substrate/BlockHandler
+        - handler: handleEvent
+          kind: substrate/EventHandler
+          filter:
+            module: balances
+            method: Deposit
+        - handler: handleCall
+          kind: substrate/CallHandler
+
+````
+
+</CodeGroupItem>
+
+
+
   <CodeGroupItem title="v0.2.0" active>
   
 ``` yml
-specVersion: 0.2.0
+specVersion: "0.2.0"
 name: example-project # Provide the project name
 version: 1.0.0  # Project version
 description: '' # Description of your project
 repository: 'https://github.com/subquery/subql-starter' # Git repository address of your project
 
 schema:
-file: ./schema.graphql # The location of your GraphQL schema file
+  file: ./schema.graphql # The location of your GraphQL schema file
 
 network:
-genesisHash: '0x91b171bb158e2d3848fa23a9f1c25182fb8e20313b2c1eb49219da7a70ce90c3' # Genesis hash of the network
-endpoint: 'wss://polkadot.api.onfinality.io/public-ws'
-
-# Optionally provide the HTTP endpoint of a full chain dictionary to speed up processing
-
-dictionary: 'https://api.subquery.network/sq/subquery/dictionary-polkadot'
+  genesisHash: '0x91b171bb158e2d3848fa23a9f1c25182fb8e20313b2c1eb49219da7a70ce90c3' # Genesis hash of the network
+  endpoint: 'wss://polkadot.api.onfinality.io/public-ws'
+  # Optionally provide the HTTP endpoint of a full chain dictionary to speed up processing
+  dictionary: 'https://api.subquery.network/sq/subquery/dictionary-polkadot'
 
 dataSources:
+  - kind: substrate/Runtime
+    startBlock: 1 # This changes your indexing start block, set this higher to skip initial blocks with less data
+    mapping:
+      file: "./dist/index.js"
+      handlers:
+        - handler: handleBlock
+          kind: substrate/BlockHandler
+        - handler: handleEvent
+          kind: substrate/EventHandler
+          filter: #Filter is optional
+            module: balances
+            method: Deposit
+        - handler: handleCall
+          kind: substrate/CallHandler
 
-- kind: substrate/Runtime
-  startBlock: 1 # This changes your indexing start block, set this higher to skip initial blocks with less data
-  mapping:
-  file: "./dist/index.js"
-  handlers: - handler: handleBlock
-  kind: substrate/BlockHandler - handler: handleEvent
-  kind: substrate/EventHandler
-  filter: #Filter is optional
-  module: balances
-  method: Deposit - handler: handleCall
-  kind: substrate/CallHandler
 
 ````
   </CodeGroupItem>
```

Comment:
Any complaints if we remove docs for 0.0.1?

Reply:
We will start drop support for 0.0.1